cmd/criticize/main.go

```diff
@@ -285,7 +286,13 @@ func (l *linter) checkFile(f *ast.File) {
 				if l.flags.ShorterErrLocation {
 					loc = shortenLocation(loc)
 				}
-				log.Printf("%s: %s: %v\n", loc, c.Rule, warn.Text)
+				log.Printf("%v:\n%v: %v\n",
+					aurora.Red(c.Rule),
+					aurora.Magenta(
+						aurora.Bold(loc),
+					),
+					warn.Text,
+				)
```

Comment:
Colored output should be optional and controlled by a flag.
It's also undesirable to break 1-line per warning rule (although some warnings are printing multi-line code, but this is a bug).
It also changes `location: checker: text` to `checker: location: text`. Not sure whether it's better.

Reply:
```txt
checker:
loc/a/tion: text
```

```txt
- ifElseChain:
/home/go/src/logger/text_formatter.go:129:2: should rewrite if-else to switch statement
```

Doesn't this formation look better?

Flag for non-colored output would be great. But also thinking about a `json` flag to work with various editor. As always `vim-go` is the first choice.

Reply:
I think it should be configurable. Having oneline output simplifies work with a log, not everyone want to see 2 or more lines per warning.
Consider to make it configurable 😉 

> But also thinking about a json flag to work with various editor.

That's a good one, probably. Do you know which other linters provides a json output?

> As always vim-go is the first choice.

Huh, debatable ;D